JenkinsFile.Promote

```diff
@@ -113,6 +113,7 @@ node {
             cf set-env ${appName} OAUTH_CLIENT_ID ${GEOAXIS_CLIENT_ID}
             cf set-env ${appName} OAUTH_SECRET ${GEOAXIS_SECRET}
             cf set-env ${appName} JKS_FILE piazza.jks
+            cf set-env ${appName} GEOAXIS_JWT_CERT GEOAXIS_JWT_CERT.jks
```

Comment:
This is really a base64 encoded certificate, so it's going to have a `.pem` extension.

Reply:
Fixed